task 3/Task 3/ShoppingListController.swift

```diff
@@ -0,0 +1,45 @@
+//
+//  ShoppingListController.swift
+//  Task 3
+//
+//  Created by Belka on 3/5/20.
+//  Copyright © 2020 Tanya. All rights reserved.
+//
+
+class ShoppingListController: ShoppingListProtocol {
+    private(set) var shoppingList = [String]()
+    init() {
+       shoppingList = ["2 carrot", "10 eggs", "Coconut milk", "Bread", "5 bananas"]
+    }
+    func add(_ item: String) -> Bool {
+        if item.isEmpty {
+            return false
+        }
+        shoppingList.append(item)
+        return true
+    }
+    
+    func remove(_ item: String) -> Bool {
+        if item.isEmpty {
+            return false
+        }
+        if let index = shoppingList.firstIndex(of: item) {
+            shoppingList.remove(at: index)
+            return true
+        }
+        return false
+    }
+    
+    func remove(at index: Int) -> Bool {
+        if index < 0 || index >= shoppingList.count  {
+            return false
+        }
+        shoppingList.remove(at: index)
+        
+        return true
+    }
+    
+    func getShoppingList() -> [String]{
```

Comment:
```suggestion
    func getShoppingList() -> [String] {
```

I saw commit where you have added SwiftLint. Probably you should improve its configuration.

Reply:
OK, thank you. I`ll configure swiftLint for this and fix it.